content/general/concepts/virtual-reality/virtual-reality.md

```diff
@@ -0,0 +1,32 @@
+---
+Title: 'Virtual Reality'
+Description: 'Virtual Reality (VR) is the computer-generated simulation of a three-dimensional image or environment that can be interacted with in a seemingly real or physical way by a person using special electronic equipment, such as a helmet with a screen inside or gloves fitted with sensors'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Best Practices'
+  - 'Memory'
+CatalogContent:
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+  - 'paths/front-end-engineer-career-path'
+---
+
+Virtual reality (VR) is a technology that uses head-mounted headsets with displays to generate realistic images, sounds, and other sensations to put users into an immersive virtual environment. VR allows us to create unbounded worlds that people can walk around and interact with using their hands, to feel as if they were transported to another place.
```

Comment:
```suggestion
Virtual reality (VR) is a technology that uses head-mounted headsets with displays to generate realistic images, sounds, and other sensations to put users into an immersive virtual environment. VR can create unbounded worlds that people can walk around and interact with using their hands, to feel as if they were transported to another place.
```

Preference is for Docs to avoid direct address to the reader in the form of using "you", "we", or "us."